easybuild/easyconfigs/r/R/R-3.5.1-intel-2018b.eb

```diff
@@ -0,0 +1,2150 @@
+    }),
+    ('tidyselect', '0.2.4', {
+        'checksums': ['5cb30e56ad5c1ac59786969edc8d542a7a1735a129a474f585a141aefe6a2295'],
+    }),
+    ('dplyr', '0.7.6', {
+        'checksums': ['d489cc8b53854ec30737bb7d39b331b67ca35f4275ad19e97420d7a247808330'],
+    }),
+    ('tidyr', '0.8.1', {
+        'checksums': ['3d3e2821c3eb711b5eebe26b880e6acfe5444d5dc69fb21761f4cd5953700511'],
+    }),
+    ('mnormt', '1.5-5', {
+        'checksums': ['ff78d5f935278935f1814a69e5a913d93d6dd2ac1b5681ba86b30c6773ef64ac'],
+    }),
+    ('foreign', '0.8-71', {
+        'checksums': ['3bd2392fbcd9dd4f385ac1418a5e454b9769e6414a21fbb5ae1e7ce6072760d7'],
+    }),
+    ('psych', '1.8.4', {
+        'checksums': ['eea18e479bf603e148d61431218f9a141f66f4ca1db5fe83d4c1f27c984dfbcf'],
+    }),
+    ('broom', '0.5.0', {
+        'checksums': ['4f912b5d9f6b7dfb24793024cdcf47e8c1a70941b907820ad4a53980771868e4'],
+    }),
+    ('nloptr', '1.0.4', {
+        'checksums': ['84225b993cb1ef7854edda9629858662cc8592b0d1344baadea4177486ece1eb'],
+    }),
+    ('lme4', '1.1-18-1', {
+        'checksums': ['982af02f33d40de88e3ec358eceec74576c8b839a526587760477e3495235905'],
+    }),
+    ('jomo', '2.6-3', {
+        'checksums': ['71ff23a7764a1fa9b94889e4180ce540ef2f83276abcee6e7881be5f24c04893'],
+    }),
+    ('hms', '0.4.2', {
+        'checksums': ['a57820b3e3221e973cba9500b4ad7953730110ee398693d150af833f26d5d0bc'],
+    }),
+    ('readr', '1.1.1', {
+        'checksums': ['1a29b99009a06f2cee18d08bc6201fd4985b6d45c76cefca65084dcc1a2f7cb3'],
+    }),
+    ('forcats', '0.3.0', {
+        'checksums': ['95814610ec18b8a8830eba63751954387f9d21400d6ab40394ed0ff22c0cb657'],
+    }),
+    ('haven', '1.1.2', {
+        'checksums': ['0a077b20bf8ef2efb73c2d4b4e1de57da7d4a525b27386320f0f7f5a9cece85e'],
+    }),
+    ('pan', '1.6', {
+        'checksums': ['adc0df816ae38bc188bce0aef3aeb71d19c0fc26e063107eeee71a81a49463b6'],
+    }),
+    ('mitml', '0.3-6', {
+        'checksums': ['bc59bdc802eb882340393752535446560c716f12c6fca2b95f03c6af30d978de'],
+    }),
+    ('mice', '3.3.0', {
+        'checksums': ['ddaf7f01f5606f3e15af05066e2f6c3ac5b6bdcc78f019e3890dab32f1ba11dc'],
+    }),
+    ('urca', '1.3-0', {
+        'checksums': ['621cc82398e25b58b4a16edf000ed0a1484d9a0bc458f734e97b6f371cc76aaa'],
+    }),
+    ('fracdiff', '1.4-2', {
+        'checksums': ['983781cedc2b4e3ba9fa020213957d5133ae9cd6710bc61d6225728e2f6e850e'],
+    }),
+    ('logistf', '1.23', {
+        'checksums': ['5adb22a40569883395dc048c877f849dd08d07582a991f1b160f0338f0b13838'],
+    }),
+    ('akima', '0.6-2', {
+        'checksums': ['61da3e556553eea6d1f8db7c92218254441da31e365bdef82dfe5da188cc97ce'],
+    }),
+    ('bitops', '1.0-6', {
+        'checksums': ['9b731397b7166dd54941fb0d2eac6df60c7a483b2e790f7eb15b4d7b79c9d69c'],
+    }),
+    ('boot', '1.3-20', {
+        'checksums': ['f339ddb7138ad05fe3e19852caf7d369f9c1ab827a70045ff66223e032cd0e06'],
```

Comment:
@smoors We had to revert the checksum update done in #8054 because the previous tarball was restored upstream (see #8271), this should be:

```
adcb90b72409705e3f9c69ea6c15673dcb649b464fed06723fe0930beac5212a
```

Reply:
fixed in 2c2e138292